src/librustc_typeck/diagnostics.rs

```diff
@@ -1041,6 +1041,7 @@ enum NightsWatch {}
 "##,
 
 E0087: r##"
+#### Note: this error code is no longer emitted by the compiler.
```

Comment:
Could you leave a blank line after the note (and the ones in the other error messages), just for readability?